pages/[address].tsx

```diff
@@ -0,0 +1,385 @@
+import React from "react";
+import styled from "@emotion/styled";
+import { GetServerSideProps, InferGetServerSidePropsType } from "next";
+import Image from "next/image";
+import { ethers } from "ethers";
+
+import {
+  Layout,
+  Hero,
+  Card as UnstyledCard,
+  Value,
+  Link as UnstyledLink,
+  About,
+  Information as UnstyledInformation,
+  SynthPlaceholderIcon,
+  GettingStarted,
+  LiveIndicator,
+  MaxWidthWrapper,
+  Table,
+} from "../components";
+import {
+  formatMillions,
+  QUERIES,
+  getContenfulClient,
+  ContentfulSynth,
+  formatContentfulUrl,
+  errorFilter,
+} from "../utils";
+import LeftArrow from "../public/icons/arrow-left.svg";
+import UnstyledRightArrow from "../public/icons/arrow-right.svg";
+import UnstyledExternalLink from "../public/icons/external-link.svg";
+import { client, EmpState, EmpStats } from "../utils/umaApi";
+
+const BackAction = () => {
+  return (
+    <ActionWrapper href="/">
+      <LeftArrow />
+      Back to projects
+    </ActionWrapper>
+  );
+};
+
+const ActionWrapper = styled(UnstyledLink)`
+  color: var(--white);
+  font-size: ${14 / 16}rem;
+  display: flex;
+  align-items: center;
+  gap: 10px;
+
+  margin-bottom: 16px;
+`;
+
+type Emp = ContentfulSynth & EmpStats & EmpState;
+export const getServerSideProps: GetServerSideProps = async (ctx) => {
+  const { address } = ctx.params as any;
+  try {
+    const contentfulClient = getContenfulClient();
+    const {
+      items: [{ fields: cmsData }],
+    } = await contentfulClient.getEntries<ContentfulSynth>({
+      content_type: "synth",
+      "fields.address": address,
+    });
+
+    const state = await client.getEmpState(address);
+    const stats = await client.getEmpStats(address);
```

Comment:
this will be gone in latest version just fyi, will need to use `tvl([address])` which will return an array `[{timestamp,value}]`

Reply:
might be able to add tvl on the emp state if you want